src/Form/_variables.scss

```diff
@@ -80,7 +84,7 @@ $custom-control-indicator-border-color: $gray-700 !default;
 $custom-control-indicator-border-width: 2px !default;
 
 $custom-control-label-color:            null !default;
-
+$form-black: $black;
```

Comment:
[curious] Is this variable used anywhere?

Reply:
No, deleted